chat-android/src/main/java/com/ably/chat/Message.kt

```diff
@@ -72,6 +75,53 @@ data class Message(
 
     /**
      * The latest action of the message. This can be used to determine if the message was created, updated, or deleted.
+     * Spec: CHA-M10
      */
     val action: MessageAction,
+
+    /**
+     * A unique identifier for the latest version of this message.
+     * Spec: CHA-M10a
+     */
+    val version: String,
+
+    /**
+     * The timestamp at which this version was updated, deleted, or created.
+     */
+    val timestamp: Long,
+
+    /**
+     * The details of the operation that modified the message. This is only set for update and delete actions. It contains
+     * information about the operation: the clientId of the user who performed the operation, a description, and metadata.
+     */
+    val operation: Message.Operation? = null,
 )
+
+fun toMessageOperation(jsonObject: JsonObject?): Message.Operation? {
```

Comment:
I think it's better to rename this function. Start top-level function's name with `to` probably isn't a good idea, I suggest `buildMessageOperation` or something that starts with the verb. Also should be internal.

Reply:
Okay, I will rename this